.github/workflows/run-release-main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         python-version: [3.9]
-        release-version: [1.1.1-post-6]
+        release-version: [1.1.1]
```

Comment:
We should think about a naming scheme for this if it's getting pushed out to pypi from the dev branch. Something like "-dev" when we push to the dev branch, and then when we create the PR from Development -> main, update the version to exclude the dev name

Reply:
We merge into dev after we make changes to make sure the changes will not break anything in pypi or read the docs then we increment versions and merge the new version into main because once a version is published in pypi from dev, you cannot publish the same version again.